src/index.js

```diff
@@ -1,11 +1,14 @@
 import React from 'react';
 import SignIn from './screens/SignIn';
 import BottomTabNav from './screens/BottomTabNav';
+import TopTabNav from './screens/TopTabNav';
+
 
 function App() {
   return(
     //<SignIn/>
-    <BottomTabNav/>
+   // <BottomTabNav/>
+    <TopTabNav/>
   );
```

Comment:
Move this to home screen. In index.js root file call navigation path where application should initialize . Follow the flow of screen

Reply:
there is no flow because we have not given navigation to different screens.. We are just commenting old screens and keeping the latest pushed screen uncommented.